configuration/messageproperties/messages_en.properties

```diff
@@ -2286,6 +2286,11 @@ ui.i18n.Form.name.8546c641-525c-474c-8696-79a60b65b6b0=Check-In
 ui.i18n.Form.name.4d778ef4-0620-11e5-a6c0-1697f925ec7b=Lab Results
 ui.i18n.Form.name.5e1b0c3a-3acf-4e50-aab1-738a3a282dea=Lab Results
 ui.i18n.Form.name.cccc5c38-eb32-11e2-981f-96c0fcb18276=Consult Provider and Date
+ui.i18n.Form.name.40f51770-0267-11e6-a837-0800200c9a66=Outpatient Intake (Adult)
+ui.i18n.Form.name.343ba950-0324-11e6-a837-0800200c9a66=Outpatient Followup (Adult)
+# ui.i18n.Form.name.40f51770-0267-11e6-a837-0800200c9a66=Outpatient Intake (Peds)
```

Comment:
I would just delete this entirely to avoid confusion in the future.

Reply:
I forgot to update the uuid in the messages files.  I'll do that now.